content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx

```diff
@@ -183,9 +183,13 @@ await prisma.$transaction([createUserOperation, createTeamOperation])
 
 ##### Are cascading deletes supported by the Prisma Client nested writes?
 
-Although you cannot specifically use _nested writes_ to perform a cascading delete (for example, delete a team and all of its members), you can still perform cascading deletes in the following ways:
+<Admonition type="warning">
 
-- Use the `$transaction` API to perform a cascading delete. Deleting a team and deleting the team's members are not dependent writes - if you know the ID of the team, you can [perform both operations within a `$transaction`](#scenario-privacy-legislation).
+In [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0) and later it is possible to do cascading deletes using the **preview feature** [referential actions](../../concepts/components/prisma-schema/relations/referential-actions). 
+
+</Admonition>
+
+You can use the `$transaction` API to perform a cascading delete. Deleting a team and deleting the team's members are not dependent writes - if you know the ID of the team, you can [perform both operations within a `$transaction`](#scenario-privacy-legislation).
```

Comment:
```suggestion
- Use the `$transaction` API to perform a cascading delete. Deleting a team and deleting the team's members are not dependent writes - if you know the ID of the team, you can [perform both operations within a `$transaction`](#scenario-privacy-legislation).
```
There is another list item beneath this one, so this would probably look weird otherwise.